src/python/pants/jvm/resolve/coursier_setup.py

```diff
@@ -80,6 +102,8 @@
     working_dir="$(pwd)"
     "$coursier_exe" fetch {repos_args} \
         --json-output-file="$json_output_file" \
+        --default=true \
+        --sources \
```

Comment:
Here's where I'm thinking it should be configurable to have coursier fetch source-jars / javadoc-jars etc. Currently I just add source-jars on top of the normal ones in this PoC

Reply:
So javadoc could be an additional type of jar added to the resolve?